pkg/generator/generator.go

```diff
@@ -118,8 +143,108 @@ func NewGenerator(apiVersion, kind, projectName, repoPath string) *Generator {
 // │   |   ├── build
 // │   |   └── codegen
 // │   └── version
+//
+// Render generates the following project structure for Operator type Ansible
+// ├── <projectName>
+// │   ├── Dockerfile
+// │   ├── roles
+// │   │   └── <kind>
+// │   ├── watches.yaml
+// │   ├── deploy
+// │   │   ├── <kind>-CRD.yaml
```

Comment:
Minort nit but unless I'm mistaken, `operator-sdk new --type=ansible --api-version=<apiversion> --kind=<kind>` will still generate:
```
// │   ├── deploy
// │   │   ├── crd.yaml
// │   │   ├── rbac.yaml
// │   │   ├── operator.yaml
// │   ├   └── cr.yaml
```

`<kind>_crd.yaml` and `<kind>_cr.yaml` are only generated by `operator-sdk generate crd --kind --api-version`.
See `renderDeployFiles()` below. Although that should still be fine I think.

Reply:
@johnkim76 is this intended? Would you prefer that we change the `new` output to have the same filenames?

Reply:
@johnkim76 based on Shawn's comment below we are going to hold off on that for the refactoring.